### PR TITLE
BLOGS-1337 Fix Mappers

### PR DESCRIPTION
**Pull request by mattgreenham**
https://jira.dev.bbc.co.uk/browse/BLOGS-1337

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
Test for this that fails before the fix and passes after?

File: tests/BlogsService/Mapper/IsiteToDomain/BlogMapperTest.php [hunk 1]
```diff
@@ -42,4 +38,25 @@ public function testGetsDomainModel()
...
+        $postMapper = new PostMapper($mapperFactory);
+        $mapperFactory->method('createPostMapper')->willReturn($postMapper);
+        $blogMapper = new BlogMapper($mapperFactory);
+
+        /** @var Blog $domainModel */
+        $blog = $blogMapper->getDomainModel($isiteObject);
+
+        $featuredPost = $blog->getFeaturedPost();
+
+        $this->assertNull($featuredPost);
```
Comment (votemike) [inline tests/BlogsService/Mapper/IsiteToDomain/BlogMapperTest.php]: Could probably just do `$this->assertNull($blog->getFeaturedPost());`

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**